### PR TITLE
Set group/module used by GraalVM annotation processor

### PR DESCRIPTION
**Pull request by ilopmar**
I open this as WIP. It needs to be merged after https://github.com/micronaut-projects/micronaut-core/pull/3367

### CODE REVIEW

**Comment by graemerocher** [comment]
PR also needs updating to support kotlin. See https://github.com/micronaut-projects/micronaut-starter/blob/master/starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw#L559

**Comment by ilopmar** [comment]
That's already included in the PR: https://github.com/micronaut-projects/micronaut-starter/pull/256/files#diff-55d171ad9926875a4ed05abf4b52e49aR567-R570

File: starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw [hunk 1]
```diff
@@ -527,13 +527,17 @@ java {
...
-tasks.withType(JavaCompile){
+tasks.withType(JavaCompile) {
     options.encoding = "UTF-8"
     options.compilerArgs.addAll([
         '-parameters',
         // enables incremental compilation
         '-Amicronaut.processing.incremental=true',
-        '-Amicronaut.processing.annotations=@project.getPackageName().*'
+        '-Amicronaut.processing.annotations=@project.getPackageName().*',
+      @if (features.contains("graalvm")) {
```
Comment (graemerocher) [inline starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw:537]: I think we can remove these checks for graalvm, this data could be useful for other processors

Comment (ilopmar) [reply starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw]: Done

**Comment by graemerocher** [comment]
Kotlin + Maven seems to be broken

**Comment by ilopmar** [comment]
You're right. I assumed (and I shouldn't) they were the same arguments as for Java. Fixed now.

**Comment by graemerocher** [comment]
Tests still failing

**Pull request merged**